documentation/asciidoc/computers/raspberry-pi/boot-eeprom.adoc

```diff
@@ -13,10 +19,13 @@ If an error occurs during boot, then an xref:configuration.adoc#led-warning-flas
 
 There are multiple ways to update the bootloader of your Raspberry Pi.
 
-==== Raspberry Pi 5, Raspberry Pi 4 and Raspberry Pi 400
+==== Flagship models since 4; Compute Modules since 5; Keyboard models since 400
 
 Raspberry Pi OS automatically updates the bootloader for important bug fixes. To manually update the bootloader or change the boot order, use xref:configuration.adoc#raspi-config[raspi-config].
 
+NOTE: Compute Module 4 and Compute Module 4S do not support automatic bootloader updates because the bootrom cannot load the `recovery.bin` file from eMMC.
+The recommended update mechanism is `rpiboot` or via `flashrom` - see `rpi-eeprom-update -h` for more information.
```

Comment:
```suggestion
NOTE: Compute Module 4 and Compute Module 4S do not support automatic bootloader updates because the bootrom cannot load the `recovery.bin` file from eMMC. The recommended update mechanism is `rpiboot` or via `flashrom` - see `rpi-eeprom-update -h` for more information.
```